src/basic-samples/ConfirmedPublish.js

```diff
@@ -126,18 +128,8 @@ var QueueProducer = function (solaceModule, queueName) {
         }
     };
 
-    producer.sendMessages = function () {
-        if (producer.session !== null) {
-            for (var i = 1; i <= producer.numOfMessages; i++) {
-                producer.sendMessage(i);
-            }
-        } else {
-            producer.log('Cannot send messages because not connected to Solace PubSub+ Event Broker.');
-        }
-    }
-
-    // Sends one message
-    producer.sendMessage = function (sequenceNr) {
+    // Create a message
+    producer.getMessage = function (sequenceNr) {
```

Comment:
Can I suggest we change the name of this function to `buildMessage`? It's more clear since its not really a getter

Reply:
Yeah sure, you are right. I will rename and push shortly!